server/adapters/dbtcloud.go

```diff
@@ -0,0 +1,167 @@
+package adapters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/jitsucom/jitsu/server/jsonutils"
+	"github.com/jitsucom/jitsu/server/templates"
+	"io/ioutil"
+	"net/http"
+)
+
+const (
+	method        = "POST"
+	urlFormat     = "https://cloud.getdbt.com/api/v2/accounts/%d/jobs/%d/run/"
+	testUrlFormat = "https://cloud.getdbt.com/api/v2/accounts/%d/jobs/%d/"
+	bodyFormat    = `{"cause": "%s"}`
+)
+
+//DbtCloudConfig is a dto for parsing DbtCloud configuration
+type DbtCloudConfig struct {
+	AccountId int    `mapstructure:"accountId" json:"accountId,omitempty" yaml:"account_id,omitempty"`
+	JobId     int    `mapstructure:"jobId" json:"jobId,omitempty" yaml:"job_id,omitempty"`
```

Comment:
I suggest using snake case in all serialization field names, like in all DTOs of such purpose.